src/libexpr/eval-settings.hh

```diff
@@ -19,11 +19,26 @@ struct EvalSettings : Config
     Setting<Strings> nixPath{
         this, getDefaultNixPath(), "nix-path",
         R"(
-          List of directories to be searched for `<...>` file references
-
-          In particular, outside of [pure evaluation mode](#conf-pure-eval), this determines the value of
+          List of search paths to use for lookup paths.
+          This setting determines the value of
           [`builtins.nixPath`](@docroot@/language/builtin-constants.md#builtins-nixPath).
-        )"};
+
+          The default value is
+
+          ```
+          $HOME/.nix-defexpr/channels
+          nixpkgs=$NIX_STATE_DIR/profiles/per-user/root/channels/nixpkgs
+          $NIX_STATE_DIR/profiles/per-user/root/channels
```

Comment:
This duplicates documentation under the `NIX_PATH` environment variable in `env-common.md`. It seems better to have this documented in only one place.

Reply:
That is true we shouldn't duplicate, but I do think this is a better place for that info, however.